view/frontend/web/js/view/payment/method-renderer/giftcards.js

```diff
@@ -64,10 +64,11 @@ define(
         }
 
         function checkPayments(){
-            var p = ["billink","afterpay","afterpay2","afterpay20","klarna","klarnakp","klarnain","capayableinstallments","sofortbanking","giropay","transfer","sepadirectdebit","capayablein3"];
+            var p = ["billink","afterpay","afterpay2","afterpay20","klarna","klarnakp","klarnain","capayableinstallments","sofortbanking","giropay","transfer","sepadirectdebit","capayablein3","creditcards","bancontactmrcash"];
```

Comment:
@saranchuk-hys why are these payment methods disabled when a Giftcard gets redeemed?

Reply:
Buckaroo currently does not support this. We are looking into this with our payment team but i advice to contact our support team via support@buckaroo.nl